cmd/app/app.go

```diff
@@ -87,6 +87,10 @@ func NewCommand(ctx context.Context) *cobra.Command {
 				}
 			}()
 
+			if !opts.DefaultTestAll {
+				log.Info("Argument test-all-containers is FALSE. Only containers with the annotation 'enable.version-checker.io' will be parsed")
```

Comment:
nit: "flag --test-all-containers=%t so only containers with the annotation 'enable.version-checker.io' will be parsed"

Reply:
Nice. I've even enriched it a bit more. Let me know if that's too much..

Reply:
No that's great! 

Can we also have the same `only containers with the annotation "enable.version-checker.io/${my-container}=true" will be parsed` for when the flag is false.

Also remove the brackets around `opts.DefaultTestAll`